datasets/bdsp-sparcnet.yaml

```diff
@@ -0,0 +1,33 @@
+Name: SPaRCNet data:Seizures, Rhythmic and Periodic Patterns in ICU Electroencephalography
+Description: The IIIC dataset includes  50,697 labeled EEG samples from 2,711 patients' and 6,095 EEGs that were annotated by physician experts from 18 institutions. These samples were used to train SPaRCNet (Seizures, Periodic and Rhythmic Continuum patterns Deep Neural Network), a computer program that classifies IIIC events with an accuracy matching clinical experts. 
+Documentation: More documentation can be found [here](https://doi.org/10.60508/cw6j-s785)
+Contact: admin@bdsp.io
+ManagedBy: "[Brain Data Science Platform](https://bdsp.io/)"
+UpdateFrequency: New data is added as soon as it is available.
+Tags:
```

Comment:
@v-mourajr can you add `aws-pds` tag to reflect the sponsorship by the AWS Open Data Program?